package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "7.50.1",
+  "version": "7.51.0",
```

Comment:
There are a couple other places where the version needs to be updated, check it out
https://github.com/workos/workos-node/pull/1275/files